client/tests/integration/smartcontracts/multisig/src/lib.rs

```diff
@@ -0,0 +1,122 @@
+//! Trigger to control multisignature account
+
+#![no_std]
+
+extern crate alloc;
+#[cfg(not(test))]
+extern crate panic_halt;
+
+use alloc::{collections::btree_set::BTreeSet, format, vec::Vec};
+
+use executor_custom_data_model::multisig::MultisigArgs;
+use iroha_trigger::{debug::dbg_panic, prelude::*};
+use lol_alloc::{FreeListAllocator, LockedAllocator};
+
+#[global_allocator]
+static ALLOC: LockedAllocator<FreeListAllocator> = LockedAllocator::new(FreeListAllocator::new());
+
+getrandom::register_custom_getrandom!(iroha_trigger::stub_getrandom);
+
+#[iroha_trigger::main]
+fn main(id: TriggerId, _owner: AccountId, event: EventBox) {
```

Comment:
```suggestion
fn main(id: TriggerId, _owner: AccountId, event: ExecuteTriggerEvent) {
```

the same. I think you can do this and conversion will be implicit